### ISSUE TITLE
Specs satisfying the dependency were found, but they required a higher minimum deployment target

### ISSUE THREAD

**Issue by Yuketsu** [mapbox/search-ios#4]
Hello!

I'm using Mapbox SDK version 6.3.0 and not able to install v1.0.0-beta.7, due to the dependency MapboxMobileEvents (~> 1.0.2) which has actual version MapboxMobileEvents (= 0.10.7, ~> 0.10.0, ~> 0.10.4). 

**Errors**:
```
In Podfile:
    MapboxSearch (< 2.0, >= 1.0.0-beta.7) was resolved to 1.0.0-beta.7, which depends on
      MapboxMobileEvents (~> 1.0.2)
Specs satisfying the `MapboxMobileEvents (= 0.10.7, ~> 0.10.0, ~> 0.10.4), MapboxMobileEvents (~> 1.0.2)` dependency were found, but they required a higher minimum deployment target.
```

**OR** 

```
In Podfile:
    MapboxSearch (< 2.0, >= 1.0.0-beta.7)
Specs satisfying the `MapboxSearch (< 2.0, >= 1.0.0-beta.7)` dependency were found, but they required a higher minimum deployment target.
```

In v1.0.0-beta.6 there is a dependency on MapboxCommon_Private and the library is not compiling. If I'm changing MapboxCommon_Private to MapboxCommon then everything works fine. However, after clearing DerivedData or pod reinstallation this change has to be made again every time.

**My Podfile**:
```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '13.0'
use_frameworks!
# ignore all warnings from all pods
inhibit_all_warnings!
workspace 'Project.xcworkspace'
target 'Project' do
	pod 'Mapbox-iOS-SDK', '~> 6.3.0'
  	pod 'MapboxSearch', ">= 1.0.0-beta", "< 2.0"
	pod 'MapboxGeocoder.swift', '~> 0.14.0'
end
```
**UPD:**
As a temporary solution it's possible to use this approach in the scheme Build Pre-actions section:
```
cd $PROJECT_DIR
sed -i '' "s/"MapboxCommon_Private"/"MapboxCommon"/" "${SRCROOT}/Pods/MapboxSearch/MapboxSearch.xcframework/ios-arm64/MapboxSearch.framework/Modules/MapboxSearch.swiftmodule/arm64-apple-ios.swiftinterface"
sed -i '' "s/"MapboxCommon_Private"/"MapboxCommon"/" "${SRCROOT}/Pods/MapboxSearch/MapboxSearch.xcframework/ios-arm64_x86_64-simulator/MapboxSearch.framework/Modules/MapboxSearch.swiftmodule/arm64-apple-ios-simulator.swiftinterface"
sed -i '' "s/"MapboxCommon_Private"/"MapboxCommon"/" "${SRCROOT}/Pods/MapboxSearch/MapboxSearch.xcframework/ios-arm64_x86_64-simulator/MapboxSearch.framework/Modules/MapboxSearch.swiftmodule/x86_64-apple-ios-simulator.swiftinterface"
```

**Comment by WonKimGoRevel** [comment]
I have the same issue.

**My Podfile**
```
pod 'Mapbox-iOS-SDK', '~> 6.3.0' 
pod 'MapboxSearch', ">= 1.0.0-beta.9", "< 2.0"
```


**Error**

```
[!] CocoaPods could not find compatible versions for pod "MapboxMobileEvents":
  In snapshot (Podfile.lock):
    MapboxMobileEvents (= 0.10.8, ~> 0.10.0, ~> 0.10.4)

  In Podfile:
    MapboxSearch (< 2.0, >= 1.0.0-beta.9) was resolved to 1.0.0-beta.9, which depends on
      MapboxMobileEvents (~> 1.0.2)

Specs satisfying the `MapboxMobileEvents (= 0.10.8, ~> 0.10.0, ~> 0.10.4), MapboxMobileEvents (~> 1.0.2)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by don1ck** [comment]
Hey Guys, Sorry late response. 
As for now with lattes maps and search there is no compatibility issue (e.g. `MapboxMaps (10.1.0)` and `MapboxSearch (1.0.0-beta.16)`)

Gonna close this issue as obsolete. Fill free to open another if you still have a problem.
